sqldelight-compiler/src/test/kotlin/app/cash/sqldelight/core/queries/PgInsertReturningTest.kt

```diff
@@ -81,8 +81,8 @@ class PgInsertReturningTest {
 
     assertThat(generator.defaultResultTypeFunction().toString()).isEqualTo(
       """
-        |public fun insertReturn(data_: com.example.Data_): app.cash.sqldelight.ExecutableQuery<com.example.Data_> = insertReturn(data_) { data__, id ->
-        |  com.example.Data_(
+        |public fun insertReturn(data_: com.example.Data_): app.cash.sqldelight.ExecutableQuery<com.example.InsertReturn> = insertReturn(data_) { data__, id ->
```

Comment:
This test verified that the code got generated, but the generated code didn't compile. I'd recommend actually deleting the test, given the integration tests have, in my opinion, better coverage, but either way, the test passes now.